### PR TITLE
Changed timestamp types to prevent bug

### PR DESCRIPTION
**Pull request by GabrielCamba**
- changed timestamp type from BigInt to Int
- removed unnecesary params on signer function
- changed DEFAULT_DECIMALS to prevent error on deploying on gnosis chain
when trying to convert it toI32()

### CODE REVIEW

File: src/mapping.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function handleTrade(event: Trade): void {
   let buyAmount = event.params.buyAmount
   let network = dataSource.network()
 
-  let timestamp = event.block.timestamp
+  let timestamp =  event.block.timestamp.toI32()
```
Comment (alfetopito) [inline src/mapping.ts:54]: Nitpick: unnecessary whitespace

```suggestion
  let timestamp = event.block.timestamp.toI32()
```

**Review by alfetopito — APPROVED** [review]
Looks ok to me.

Could you explain what bug is this timestamp type change fixing?

**Comment by GabrielCamba** [comment]
> Looks ok to me.
> 
> Could you explain what bug is this timestamp type change fixing?

We found timestamps were not rounded as they were on uniswap V3. The only difference between they subgraph and ours was the type. Chaging the type the not rounded error was solved.

**Review by anxolin — APPROVED** [review]
